PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,4 +1,5 @@
 Fixes #issue number
 
 Check everything which applies
+
 [] I have added the issue number for which this pull request is created.
```

Comment:
Change this to
`- [ ] I have added the issue number for which this pull request is created.`

Copy the exact line and paste it